v4/weaviate/graphql/neartextbuilder.go

```diff
@@ -1,11 +1,72 @@
 package graphql
 
 import (
-	"encoding/json"
 	"fmt"
 	"strings"
 )
 
+// fldMover is a type representing field names of a move sub query
+type fldMover string
+
+const (
+	fldMoverConcepts fldMover = "concepts"
+	fldMoverForce    fldMover = "force"
+	fldMoverID       fldMover = "id"
+	fldMoverBeacon   fldMover = "beacon"
+	fldMoverObjects  fldMover = "objects"
+)
+
+// MoveParameters to fine tune Explore queries
+type MoveParameters struct {
+	// Concepts that should be used as base for the movement operation
+	Concepts []string
+	// Force to be applied in the movement operation
+	Force float32
+	// Movers is a list of movers used to adjust the serach direction
+	Movers []MoverObject
+}
+
+func (m *MoveParameters) String() string {
+	concepts := marshalStrings(m.Concepts)
+	ms := make([]string, 0, len(m.Movers))
+	for _, m := range m.Movers {
+		if s := m.String(); s != EmptyObjectStr {
+			ms = append(ms, m.String())
+		}
+	}
+	if len(ms) < 1 {
+		return fmt.Sprintf("{%s: %s %s: %v}", fldMoverConcepts, concepts, fLdMoverForce, m.Force)
+	}
+
+	s := "{"
+	if len(m.Concepts) > 0 {
+		s = fmt.Sprintf("{%s: %s", fldMoverConcepts, concepts)
```

Comment:
I think that we are missing here a closing bracket?

Reply:
The "}" is included in the next line